package.json

```diff
@@ -7,16 +7,7 @@
     "type": "git",
     "url": "git+https://github.com/FaridSafi/react-native-gifted-chat.git"
   },
-  "keywords": [
-    "android",
-    "ios",
-    "react-native",
-    "react",
-    "react-component",
-    "messenger",
-    "message",
-    "chat"
-  ],
+  "keywords": ["android", "ios", "react-native", "react", "react-component", "messenger", "message", "chat"],
```

Comment:
Why this change? Generally it's nice to keep things on separate lines for the purpose of tracking history.

Reply:
Sorry, will revert.